src/shipit_code_coverage/shipit_code_coverage/chunk_mapping.py

```diff
@@ -125,6 +125,14 @@ def get_test_coverage_files(tests):
     return r.json()['data']
 
 
+def should_ignore_suite(suite):
```

Comment:
Let's rename this to something like `is_chunk_suite` or `is_chunk_only_suite` (making it the opposite of what it is now), and update its usage to add `not`